torch/_inductor/kernel/flex_attention.py

```diff
@@ -845,6 +870,9 @@ def flex_attention(
             (64, 64, 4, 3),
         ]
 
+        # On ROCm convert num_stages to 1 to avoid shmem issues
+        configs = [(c[0], c[1], c[2], 1) for c in configs]
+
```

Comment:
Where is this config change guarded to only apply to ROCm?

Reply:
Ah good find, I guarded it in flex_decode but I missed it out here... should be this:
```
  if torch.version.hip:
      configs = [(c[0], c[1], 1) for c in configs]
```